jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -8521,6 +8524,14 @@ public String translateEscapes() {
 					}
 					builder.append((char)octal);
 					break;
+				case '\r':
```

Comment:
I am not sure if this `\r` case is required as all newline should be normalized to `\n` during `stripIndent()` call before this method is called. (line 8441 in this file)

Reply:
The `verifyLineTerminator()` test from [TranslateEscapes.java](https://github.com/ibmruntimes/openj9-openjdk-jdk14/blob/openj9/test/jdk/java/lang/String/TranslateEscapes.java) failed on `\r` and `\r\n` when I didn't have the case for `\r`. I think this is because the test directly calls `translateEscapes()`, so the string isn't normalized by `stripIndent()`